client/src/components/Tool/ToolCard.vue

```diff
@@ -70,6 +70,12 @@ const props = defineProps({
         type: String,
         default: null,
     },
+    /** Whether there will be a `FormDisplay` rendered in the default slot with all tool inputs.
+     * For now, this is used to determine if tool generated tours can be offered. */
+    hasCompleteInputs: {
```

Comment:
I see the motivation, but I would keep it specific and just use `allowGeneratedTours` as above.